.github/workflows/test_app.yml

```diff
@@ -18,6 +18,26 @@ on:
         description: 'The testing command to be ran'
         required: true
         type: string
+      bullet_enabled:
+        description: 'Enable bullet performance testing'
+        required: false
+        default: false
+        type: boolean
+      bullet_n_plus_one:
+        description: 'Enable bullet n+1 testing'
+        required: false
+        default: false
+        type: boolean
+      bullet_counter_cache:
+        description: 'Enable bullet counter cache testing'
+        required: false
+        default: false
+        type: boolean
+      bullet_unused_eager_loading:
+        description: 'Enable bullet unused eager loading testing'
+        required: false
+        default: false
+        type: boolean
```

Comment:
I was just thinking do we need to be so granular with these CI options if we only have two cases right now:
1. All enabled
2. All disabled

I would maybe only use `bullet_enabled` and then apply that to the workflows where we are using a value other than the default.

I like the granularity with the ENV vars so that we can locally test only one case at a time but for the CI I would consider consolidating these into one option.

Reply:
I have the same thought, but considering that we will start the process of query optimization, maybe it would be ok to focus just on a single topic. The way i have added would allow me to focus on various topics ( like remove the n+1) queries from core.